lib/compiler.js

```diff
@@ -820,3 +820,12 @@ function normalizeMixinName(str, options) {
       throw err;
   }
 }
+
+function fileExists(path) {
+  try {
+    fs.statSync(path);
+    return true;
+  } catch (e) {
+    return false;
+  }
+}
```

Comment:
I thought about that but didn't at first because the logic was only used more than once in  `compiler.js`, and the function it's used in in `config-loader.js`, and there wasn't already a `util` file for shared logic.

Anyhow, I'm happy to do move it. I'll do it now.